### ISSUE TITLE
Angular 11 Support

### ISSUE THREAD

**Issue by MaxKless** [DethAriel/ng-recaptcha#203]
Hi, currently ng-recaptcha supports angular versions 6 through 10. Is support for Angular 11 coming? Are there any issues that you are aware of?

**Comment by gogakoreli** [comment]
After migrating project to v11 I had random errors, which got fixed by disabling ivy via tsconfig's "enableIvy": false flag.
After spending a lot of time, figured out that it was due to ng-recaptcha not supporting v11 yet. I confirmed the issue by removing all of the reference of ng-recaptcha usage, while having ivy enabled and --prod build passing.

I was importing ng-recaptcha via: RecaptchaModule.forRoot() and forRoot() is deprecated api. By removing forRoot() and simply importing module, those random errors got fixed and --prod build passed with ivy enabled. Hope this helps someone else looking for solutions. Tbh, I am not sure how it fixed the issue.

In the meantime this fixed my issue and ng-recaptcha works fine with Angular v11. Hope to see Angular v11 supported by you officially, thus making warnings disappear as well. Thanks for all your effort on the component.
